modules/__tests__/Router-test.js

```diff
@@ -272,7 +268,7 @@ describe('Router', function () {
         render() {
           const { label, children } = this.props
           const child = React.cloneElement(children, {
-            createElement: this.createElement
+            createElement: this.createElement.bind(this)
```

Comment:
Awesome, thanks for the explanation. And your help in general. (Happy
friday everyone!).

On Fri, Nov 13, 2015 at 3:24 PM, Jimmy Jia <notifications@github.com> wrote:

> In modules/__tests__/Router-test.js
> <https://github.com/rackt/react-router/pull/2540#discussion_r44829782>:
>
> > @@ -272,7 +268,7 @@ describe('Router', function () {
> >          render() {
> >            const { label, children } = this.props
> >            const child = React.cloneElement(children, {
> > -            createElement: this.createElement
> > +            createElement: this.createElement.bind(this)
>
> Yes - it breaks referential equality, which leads to e.g. pure render not
> working. It seldom ever really matters, but it's just good hygiene.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/rackt/react-router/pull/2540/files#r44829782>.
>